orchestrator-salt/src/main/java/com/sequenceiq/cloudbreak/orchestrator/salt/client/TracingClientSpanDecorator.java

```diff
@@ -0,0 +1,33 @@
+package com.sequenceiq.cloudbreak.orchestrator.salt.client;
+
+import javax.ws.rs.client.ClientRequestContext;
+import javax.ws.rs.client.ClientResponseContext;
+
+import org.apache.commons.lang3.StringUtils;
+
+import com.sequenceiq.cloudbreak.orchestrator.salt.domain.Pillar;
+import com.sequenceiq.cloudbreak.tracing.TracingUtil;
+
+import io.opentracing.Span;
+import io.opentracing.contrib.jaxrs2.client.ClientSpanDecorator;
+
+public class TracingClientSpanDecorator implements ClientSpanDecorator {
+
+    @Override
+    public void decorateRequest(ClientRequestContext requestContext, Span span) {
+        String uri = requestContext.getUri().toString();
+        String saltPath = StringUtils.substringAfterLast(uri, "/saltboot/");
```

Comment:
most salt calls happens using `/saltapi/run`. see `SaltEndpoint#SALT_RUN`
this way `StringUtils.substringAfterLast` will return an empty string

Reply:
this explains an anomely I just saw in jaeger